package.json

```diff
@@ -21,16 +21,16 @@
   },
   "homepage": "https://github.com/GSG-K3/code-academy-wiki#readme",
   "dependencies": {
+    "axios": "^0.19.2",
     "concurrently": "^5.1.0",
     "dotenv": "^8.2.0",
     "env": "0.0.2",
     "env2": "^2.2.2",
     "express": "^4.17.1",
     "includes": "0.0.7",
     "nodemon": "^2.0.3",
-    "pg": "^8.0.3",
+    "pg": "^7.18.2",
```

Comment:
@Rahaf-96 Did you intend to commit this?  It looks like it downgrades `pg` from 8.0.3 to 7.18.2

Reply:
We had an error called self signed certificate, so when we searched for a solution we found out that changing pg version solves the problem.
Actually there was another solution which is adding this line to the code: NODE_TLS_REJECT_UNAUTHORIZED='0'
But we think that the better one is changing pg version.

Reply:
Ahh okay, thanks for the explanation!